src/python/pants/help/help_printer.py

```diff
@@ -133,10 +132,13 @@ def _print_options_help(self) -> None:
             print(pants_release())
             print("\nUsage:")
             print(
-                f"  {self.bin_name} [option ...] [goal ...] [target...]  Attempt the specified goals."
+                f"  {self.bin_name} [option ...] [goal ...] [target/fileglob ...]  Attempt the specified goals."
```

Comment:
Do we want to call this fileglob or file arg? I don't have a strong opinion either way.